function/inception.py

```diff
@@ -585,5 +585,5 @@ def invoke(body):
 ########################################################################
 
 if __name__ == '__main__':
-	print(invoke())
+	print(invoke(os.path.join(data_dir, 'cropped_panda.jpg')))
```

Comment:
Why did this change get made?

Reply:
when testing inception.py to make sure tensorflow worked, i wanted to run 'python3 inception.py', but the old version didn't supply a body arg so it crashed.